.github/workflows/test.yml

```
@@ -157,7 +157,17 @@ jobs:
         timeout-minutes: 10
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: make testacc
+        run: make testacc-cover
+
+      - name: Upload coverage report to Codecov
+        if: matrix.flavor.target == 'linux'
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)